beacon_node/http_api/src/lib.rs

```diff
@@ -2923,7 +2923,13 @@ pub fn serve<T: BeaconChainTypes>(
                         match chain.verify_aggregated_attestation_for_gossip(aggregate) {
                             Ok(verified_aggregate) => {
                                 messages.push(PubsubMessage::AggregateAndProofAttestation(Box::new(
-                                    verified_aggregate.aggregate().clone(),
+                                    LazySignedAggregateAndProof::from_ssz_bytes(&verified_aggregate.aggregate().clone().as_ssz_bytes()).map_err(|e| {
```

Comment:
I figured there was a better way to do this like you say, but I can't seem to find the method to directly compress the signature. This is the best I've found `SignatureBytes::from_ssz_bytes(&self.signature.as_ssz_bytes())?`. Is there a better way?

Reply:
There's a `From` impl that you can use on line 94 here: https://github.com/sigp/lighthouse/blob/stable/crypto/bls/src/generic_signature_bytes.rs

It does basically the same thing though

Reply:
I used the way I had written previously. I tried to use From and it wasnt working for some reason. I think I implemented the changes though

Reply:
Hmm... I can fix the clippy error, but it doesn't seem preferable to me. The other failures aren't occurring on my computer so I'm not sure whats going on there but I'll try to figure it out.

Reply:
That's ok, I think the other errors are just spurious failures. We still have problems with `Address already in use` due to race conditions (the Linux failure), and the Windows error looks like a download fail

Reply:
Ah yeah, the `From` doesn't work because we've got an `AggregateSignature` and there's no logic for converting an `AggregateSignature` to a `Signature`/`SignatureBytes`. Lets leave it as-is.